.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+  // Attach to Air for debugging
```

Comment:
should this comment be in JSON

Reply:
Good callout. Not sure if https://github.com/air-verse/air supports JSON5

Reply:
actually maybe I should remove this and ignore all of .vscode directory